logging/google/cloud/logging/client.py

```diff
@@ -40,6 +41,12 @@
 from google.cloud.logging.connection import _LoggingAPI as JSONLoggingAPI
 from google.cloud.logging.connection import _MetricsAPI as JSONMetricsAPI
 from google.cloud.logging.connection import _SinksAPI as JSONSinksAPI
+from google.cloud.logging.handlers import CloudLoggingHandler
+from google.cloud.logging.handlers import AppEngineHandler
+from google.cloud.logging.handlers import ContainerEngineHandler
+from google.cloud.logging.handlers import setup_logging
+from google.cloud.logging.handlers.handlers import EXCLUDED_LOGGER_DEFAULTS
```

Comment:
The circular dependency gets created between `handlers.__init__.py` and `handlers.handlers.py`